python/shotgun_desktop/startup.py

```diff
@@ -313,6 +313,18 @@ def _quit(self):
         """
         self.exit(self.CLOSE_APP)
 
+    def exec_(self):
+        """
+        Execute the local event loop. If CmdQ was hit in the past, it will be handled just as if the
+        user had picked the Quit menu.
+        """
+        code = QtCore.QEventLoop.exec_(self)
+        # CmdQ was hit before.
```

Comment:
True, howdo technically ALT-F4 doesn't meant close the application on Windows it means close the current window.